src/cr/cube/matrix/measure.py

```diff
@@ -1899,24 +1919,22 @@ def _opposing_numeric_values(self):
         )
 
 
-class _MarginProportion(_BaseMarginal):
-    """The 'proportion-margin' (aka rows-margin-proportion/columns-margin-proportion)"""
+class _MarginTableProportion(_BaseMarginal):
+    """The 'margin-table-proportion', proportions of the margin (as if it were a stripe)
+
+    Defined as the margin-weighted-counts divided by the margin-weighted-base.
+    """
```

Comment:
Hmm, okay, so this bit about the "table" being in the name has to do with the stripe-like idea? We probably need to give that some further thought or rethinking. Happy to chat about that if you like.